Jenkinsfile

```diff
@@ -11,45 +11,36 @@ pipeline {
 
     options {
         ansiColor('xterm')
-        disableConcurrentBuilds()
```

Comment:
I see PORT is static, so might be a bit of a problem. I remember when I made this pipeline I was starting a cluster on a "free" port, but it could have changed.

Reply:
Since we are starting a new agent for each build, we can just make use of a static port, instead of trying to look for a free one.